### ISSUE TITLE
Spurious lint error in cljs for `parse-uuid`

### ISSUE THREAD

**Issue by harold** [candid82/joker#489]
Yo! Thanks for joker. Still linting hundreds of files every day in real time with everything hooked to flycheck in emacs. This project has certainly saved me _tons_ of time over the years, and I am very grateful to you and everyone for all the efforts.

```
harold@freeside:~$ cat test.cljs
(ns test)

(parse-uuid "266c2d5c-47f4-4c61-a3bb-64c2bd08accc")
harold@freeside:~$ joker --version
v1.3.0
harold@freeside:~$ joker --lint test.cljs
test.cljs:3:2: Parse error: Unable to resolve symbol: parse-uuid
```

`parse-uuid` is in cljs now

I'm not an expert, but maybe it just needs to be added near here (?):

https://github.com/candid82/joker/blob/6621de3d0160f363f16717ce63ee0f4f794b6ab7/core/data/linter_cljs.joke#L70

**Comment by candid82** [comment]
@harold thank you for the kind words! The issue is fixed by https://github.com/candid82/joker/commit/1d154363b529976a0aeec74cededf6147f218fe5. I'll release a new version soon.

**Comment by harold** [comment]
Yay! Thanks. Stay rad.

**Comment by candid82** [comment]
Fixed in v1.3.1.
